FileHandlingAssignment/deleteBookList.php

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Books list To Delete</title>
+</head>
+<body>
+	<h1>Welcome : 
+	<?php
+		 include 'config.php';
+		 session_start(); echo "".$_SESSION['userName'];
+	?>
+	</h1>
+	<hr>
+	<h2>
+	<form name="bookList" method="POST" action="deleteBook.php">
+	<?php 
+			$sqlQuery= "SELECT bookName FROM books_details WHERE user_name = '".$_SESSION['userName']."'"; 
```

Comment:
I've asked you all to keep the DB name, DB columns and table names underscore_separated.